### PR TITLE
Upgrade gqlgen library

### PR DESCRIPTION
**Pull request by la4ezar**
**Description**
Currently, we are using a very old version of the gqlgen library which refers to some packages that no longer exist. With this PR we upgrade the gqlgen library to the latest version.

Changes proposed in this pull request:
- Upgrade gqlgen library

**Pull Request status**

- [ ] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [ ] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: tests/system-fetcher/tests/handler_test.go [hunk 1]
```diff
@@ -1544,7 +1544,7 @@ func waitForApplicationsToBeProcessed(ctx context.Context, t *testing.T, tenantI
 		_, actualApps := retrieveAppsForTenant(t, ctx, tenantID)
 		t.Logf("Found %d from %d", len(actualApps), expectedNumber)
 		return len(actualApps) >= expectedNumber
-	}, time.Minute*1, time.Second*1, "Waiting for Systems to be fetched.")
+	}, time.Second*90, time.Second*1, "Waiting for Systems to be fetched.")
```
Comment (ivantenevvasilev) [inline tests/system-fetcher/tests/handler_test.go:1547]: Why was this needed?

**Review by ivantenevvasilev — APPROVED** [review]

**Review by emmakarova — APPROVED** [review]

**Review by georgifarashev — APPROVED** [review]
Ok for IAS adapter
